services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -228,6 +229,19 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
         initialValues={initialValues}
       >
         <Row gutter={[16, 8]}>
+          {system === SystemFlagEnum.core && (
+            <Col span={24}>
+              <Typography.Paragraph>What is the type of the report?</Typography.Paragraph>
```

Comment:
Would recommend using the RenderRadio button instead of manually making it (and having the typography/required indicator not match...). Not keen on using 1/2 for values, hard to parse when you're debugging.

Reply:
Thanks for the recommendation, this should be adjusted now.